kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -258,6 +258,8 @@ def check(self, instance):
         # Kubelet credentials handling
         self.kubelet_credentials.configure_scraper(self.cadvisor_scraper_config)
         self.kubelet_credentials.configure_scraper(self.kubelet_scraper_config)
+        self.get_http_handler(self.cadvisor_scraper_config, recreate=True)
```

Comment:
I think it would be cleaner/safer to just clear `self._http_handlers` at the beginning of the check. WDYT?